Mathlib/Tactic/Slice.lean

```diff
@@ -49,25 +55,25 @@ def evalSlice (a b : Nat) : TacticM Unit := do
   let _ ← iterateUntilFailureWithResults do
     evalTactic (← `(conv| rw [Category.assoc]))
 
-/-- `slice` is implemented by `evalSlice` -/
-elab "slice" a:num b:num : conv => evalSlice a.getNat b.getNat
+/-- `slice` is implemented by `evalSlice`. -/
+elab "slice " a:num b:num : conv => evalSlice a.getNat b.getNat
 
-/-- 
-`sliceLHS a b => tac` is a conv tactic which enters the LHS of a target, 
-uses `slice` to extract the `a` through `b` terms, and then applies 
-`tac` to the result. 
+/--
+`slice_lhs a b => tac` zooms to the left hand side, uses associativity for categorical
+composition as needed, zooms in on the `a`-th through `b`-th morphisms, and invokes `tac`.
 -/
-syntax (name := sliceLHS) "sliceLHS" num num " => " convSeq : tactic
+syntax (name := sliceLHS) "slice_lhs " num num " => " convSeq : tactic
```

Comment:
This conflicts with the naming convention for tactics in Lean core. Eg. [Lean.Init.Tactics](https://leanprover-community.github.io/mathlib4_docs/Init/Tactics.html). Though mathlib seems to have not adhered to any one convention...

Reply:
For porting tactics, we use the syntax given in [`Mathlib/Mathport/Syntax.lean`](https://github.com/leanprover-community/mathlib4/blob/ecd37441047e490ff2ad339e16f45bb8b58591bd/Mathlib/Mathport/Syntax.lean#L450), which is what `Mathport` uses!

Also I think the convention for the actual call syntax is lower_snake_case, judging by the syntax for many tactics in that file? I'm not sure if it's written down somewhere though.

Reply:
Ah you are right. I am confusing the name with the syntax. Thanks!